client/containers/DashboardEdges/NewEdgeInput.js

```diff
@@ -42,6 +42,7 @@ const indeterminateMenuItem = (
 		isSkeleton={true}
 		showImage={true}
 		disabled={true}
+		onClick={() => {}}
```

Comment:
This is just to fix a PropType error that was preventing any other logging from appearing in the Jest runner /shrug